Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM node:10 as builder
 
 ARG KOMENCI_VERSION
```

Comment:
Is this first `ARG KOMENCI_VERSION` needed?

Reply:
I'm actually not sure, hmmm let's try without it and see if it works 😂

Reply:
Just tested this in alfajores and it worked well :) Merging this PR